### PR TITLE
fixes bug 1189392 - remove socorro.external.elasticsearch

### PR DESCRIPTION
**Pull request by peterbe**
@AdrianGaudebert r?

### CODE REVIEW

**Comment by peterbe** [comment]
Because it touches CrashStorage stuff, do you want to wait till @twobraids is back?
I think getting rid of `socorro.external.elasticsearch` is crucial for me to be able to finish my work on moving the super_search_fields from middleware to the webapp.

**Comment by AdrianGaudebert** [comment]
That awkward moment when a colleague asks you to review a patch that removes 6.5k lines of code you wrote. :D

**Comment by peterbe** [comment]
HAHA! Yeah. It's weird. But work. :)

**Comment by peterbe** [comment]
@AdrianGaudebert tests pass :)

**Comment by peterbe** [comment]
Adrian, maybe we'll wait with merging this til *after* we've done a release. There's too many other things going on at the moment.

File: config/crontabber.ini-dist [hunk 1]
```diff
@@ -428,7 +428,7 @@
             #forbidden_keys=url, email, user_id, exploitability,json_dump.sensitive,upload_file_minidump_flash1.json_dump.sensitive,upload_file_minidump_flash2.json_dump.sensitive,upload_file_minidump_browser.json_dump.sensitive,memory_info
 
             # None
-            #index_creator_class=socorro.external.elasticsearch.crashstorage.ElasticSearchCrashStorage
+            #index_creator_class=socorro.external.es.crashstorage.ElasticSearchCrashStorage
```
Comment (AdrianGaudebert) [inline config/crontabber.ini-dist]: The new one is called ``ESCrashStorage``.

File: config/test_automatic_emails.ini-dist [hunk 2]
```diff
@@ -104,7 +104,7 @@ exacttarget_user=
     # doc: None
     # Inspect the automatically written value below to make sure it is valid
     #   as a Python object for its intended converter function.
-    #index_creator_class='socorro.external.elasticsearch.crashstorage.ElasticSearchCrashStorage'
+    #index_creator_class='socorro.external.es.crashstorage.ElasticSearchCrashStorage'
```
Comment (AdrianGaudebert) [inline config/test_automatic_emails.ini-dist]: This thing doesn't exist anymore, you can remove this entire file! :)

File: docs/development/api/crashstorage.rst [hunk 3]
```diff
@@ -298,15 +298,6 @@ socorro.external.rabbitmq.rmq_new_crash_source
 A pluggable Functor/generator for feeding new crashes to the processor,
 implemented as a wrapper around new_crashes().
 
-socorro.external.elasticsearch
-------------------------------
-
-TBD by adrian.
```
Comment (AdrianGaudebert) [inline docs/development/api/crashstorage.rst]: Wow haha I never saw this before... Guess I kind of didn't do it... :)

File: scripts/move_emails.py [hunk 4]
```diff
@@ -31,14 +31,14 @@ class MoveEmailsApp(generic_app.App):
     required_config.namespace('elasticsearch')
     required_config.elasticsearch.add_option(
         'elasticsearch_class',
-        default='socorro.external.elasticsearch.connection_context.'
+        default='socorro.external.es.connection_context.'
```
Comment (AdrianGaudebert) [inline scripts/move_emails.py]: Same here, this file was a one-time script, we won't ever need it again, feel free to remove it.

**Comment by AdrianGaudebert** [comment]
A few more files that can be removed and then we're good! 

r+

**Pull request merged**